modules/element-manager/assets/js/app.js

```diff
@@ -590,7 +598,8 @@ export const App = () => {
 									</FlexItem>
 									<FlexItem>
 										<UpgradeButton
-											href="https://go.elementor.com/go-pro-element-manager/"
+											href={ elementManager.url }
+											text={ elementManager.text }
```

Comment:
@davseve Should you not update the `UpgradeButton` component as well?

In `elementor/modules/element-manager/assets/js/upgrade-button.js` the text is still hard coded and not using the props.

Reply:
The hard-coded text has been removed, by passing the `text` prop I update the text inside the button component which the `update_button` inherits from.

I've tested this manually and it works.

Reply:
@davseve I did not see the changes in `upgrade-button.js`, maybe I missed it, sorry.